changes/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+
+from .models import Change
+
+admin.site.register(Change)
```

Comment:
please do a custom admin with all the `registrant_id` `action` and `validated`  fields on display to help with management pre CI